rmi/servercode/ResImpl/RMHashtable.java

```diff
@@ -28,11 +29,30 @@ public String toString()
 		}
 		s = s + "--- END RMHashtable ---";
 		return s;
-  	}
+  }
 
-  	public void dump()
-	{
+
+  public void dump() {
 		System.out.println( toString() );
-  	}
+  }
+
+

```

Comment:
below we're writing/storing an RMHashtable into a file, named `filename` under the shadows/ directory